src/components/dev-hub/card-badge.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { colorMap, size } from './theme';
+import { P5 } from './text';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+const CardBadge = styled('div')`
+    align: left;
```

Comment:
What is this css property doing?

Reply:
I thought this was responsible for positioning the div towards the left, but it looks like it does nothing. I removed it.